### ISSUE TITLE
tree update functionality broken on mobx-angular@2.0.0

### ISSUE THREAD

**Issue by antoantonyk** [500tech/angular-tree-component#491]
after installing angular-tree-component@6.0.0,  this.tree.treeModel.update(); seems broken. 

Package.json
```
{
  "name": "axx-xxx",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "serve": "ng serve --proxy-config proxy.conf.json",
    "build": "ng build",
    "build_dev": "ng build --output-path=../src/main/resources/public",
    "build_prod": "ng build --prod --output-path=../src/main/resources/public",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "compodoc": "./node_modules/.bin/compodoc -p src/tsconfig.app.json"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "5.0.5",
    "@angular/common": "5.0.5",
    "@angular/compiler": "5.0.5",
    "@angular/core": "5.0.5",
    "@angular/forms": "5.0.5",
    "@angular/http": "5.0.5",
    "@angular/platform-browser": "5.0.5",
    "@angular/platform-browser-dynamic": "5.0.5",
    "@angular/router": "5.0.5",
    "@ng-bootstrap/ng-bootstrap": "1.0.0-beta.6",
    "@ultimate/ngxerrors": "1.3.0",
    "angular-tree-component": "^6.0.0",
    "bootstrap": "4.0.0-beta.2",
    "bootstrap-material-design": "0.5.10",
    "core-js": "2.5.1",
    "font-awesome": "4.7.0",
    "jquery": "3.2.1",
    "ngx-progressbar": "2.0.8",
    "ngx-toastr": "6.2.1",
    "popper.js": "1.12.3",
    "rxjs": "5.5.3",
    "web-animations-js": "2.3.1",
    "zone.js": "0.8.18"
  },
  "devDependencies": {
    "@angular/cli": "1.5.5",
    "@angular/compiler-cli": "5.0.5",
    "@angular/language-service": "5.0.5",
    "@compodoc/compodoc": "1.0.5",
    "@types/jasmine": "2.8.2",
    "@types/jasminewd2": "2.0.3",
    "@types/node": "6.0.92",
    "codelyzer": "4.0.1",
    "jasmine-core": "2.6.2",
    "jasmine-spec-reporter": "4.1.0",
    "karma": "1.7.0",
    "karma-chrome-launcher": "2.1.1",
    "karma-cli": "1.0.1",
    "karma-coverage-istanbul-reporter": "1.2.1",
    "karma-jasmine": "1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "protractor": "5.1.2",
    "ts-node": "3.2.2",
    "tslint": "5.7.0",
    "typescript": "2.4.2"
  }
}

```

Plunker: https://plnkr.co/edit/XrVm5YWmNSfQoW7JA0L6?p=options

as a workaround i installed the mobx-angular@1.9.0 to fix this issue.

**Comment by ivucicev** [comment]
Suddenly facing the same issue.

**Comment by adamkleingit** [comment]
I see - latest mobx-angular version detaches from change detection.
Will be fixed in 6.0.1


*Adam Klein*
*Chief Technology Officer*



*Tel.*   +972-52-7475633 <+972527475633>
*Web*  500tech.com <http://www.500tech.com/> | github.com/500tech |
@adamklein500 <http://twitter.com/adamklein500>

**Comment by adamkleingit** [comment]
Fixed in 6.1.0
